docs/adguard-vpn-for-windows/solving-problems/common-installer-errors.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Common installer errors
+sidebar_position: 2
+---
+
+This article contains some of the most common errors that you can encounter during the AdGuard VPN for Windows installation, and possible ways to solve them.
+
+### Error 5: Access Denied {#error-5}
+
+This error occurs when there's something wrong with permissions. There may be several different reasons why AdGuard VPN installer doesn't have the permissions it requires to properly finish the installation process. You can try the following steps:
```

Comment:
also, try to abstain from using apostrophes in the Knowledge base (there are six instances of them, if I'm not mistaken)

Reply:
hm, okay, I didn't write this original text tho :)